build-system/pr-check/visual-diff-tests.js

```diff
@@ -30,21 +31,23 @@ const {runCiJob} = require('./ci-job');
 
 const jobName = 'visual-diff-tests.js';
 
+const esmFlag = argv.type === 'module' ? ' --esm' : '';
```

Comment:
For better readability and maintainability, can we replace this line with the following, and use `baseCommand` in the commands below?
```suggestion
let baseCommand = 'gulp visual-diff';
if (argv.type === 'module') {
  baseCommand += ' --esm';
}
```